src/functions/blogMethods/transform-custom-mdx.js

```diff
@@ -0,0 +1,18 @@
+const { default: Map } = require('apr-map')
+const Waterfall = require('apr-waterfall')
+
+const TransformImageData = require('./transform-image-data')
+const TransformIFrames = require('./transform-iframes')
+const TransformStrings = require('./transform-strings')
+const AddFrontMatter = require('./add-frontmatter.js')
+
+module.exports = async (posts, environment) => {
+  return Map(posts, async post =>
```

Comment:
Some modules export with `Map`, some without

Reply:
move index Map methods to blogMethods/